BuildingBlocks/src/Tooling/Extensions/ILoggerExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Diagnostics;
+using Enmeshed.Tooling.Extensions;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Logging;
+
+public static class ILoggerExtensions
+{
+    private const string DEBUG_PERFORMANCE_ENV_VAR = "DEBUG_PERFORMANCE";
+    private static readonly EventId EVENT_ID_EXECUTION_TIME = new(1000, "ExecutionTime");
+
+    public static async Task TraceTime(this ILogger logger, Func<Task> action, string? actionName = null)
+    {
+        if (Environment.GetEnvironmentVariable(DEBUG_PERFORMANCE_ENV_VAR).IsNullOrEmpty())
```

Comment:
Can you extract this into a static class + variable please, so that we can use it in other places as well?

And remember to replace all usages of this with the new class.

```c#
public static class EnvironmentVariables
{
    public static readonly string DEBUG_PERFORMANCE = Environment.GetEnvironmentVariable("DEBUG_PERFORMANCE");
}